README.md

```diff
@@ -1 +1,84 @@
-# atlas_starter_go
\ No newline at end of file
+# MongoDB Atlas Golang Sample Project
+
+This repository contains an example application that connects to MongoDB
+Atlas using the Go MongoDB driver. You can use this application as
+a reference for when you build your Golang application.
+
+
+## Prerequisites
+
+To build and run this project, you need Golang version 1.13.x or later.
+
+
+## Getting Started
+
+The following instructions explain how to get your project connected to your 
+instance of MongoDB Atlas.
+
+### 1. Download the Repository
+
+To get started with this sample project, download this repository to your
+programming environment. You can clone this project using Git version control:
+
+```
+git clone https://github.com/mongodb-university/atlas_starter_go.git
+```
+
+Or you can download the ZIP archive using your browser. If you download
+this project as a ZIP archive, 
+[unzip the archive](https://www.wikihow.com/Unzip-a-File) before proceeding.
+
+
+### 2. Install Dependencies
+
+Navigate to the directory containing the project in your shell or open it
+in your preferred IDE.
+
+Then, install the necessary packages by running the following command.
+
+```shell
+go get -d
+```
+
+
+### 3. Configure your Atlas Credentials
+
+1. Open the  `main.go` file.
+
+2. Search for the variable `mongoUri` near the top which is assigned
+   placeholder text. Replace the placeholder text with the connection
+   string for your Atlas cluster. For more information on finding the
+   connection string, see [Connect via
+   Driver](https://docs.atlas.mongodb.com/driver-connection/) in the Atlas
+   documentation.
+
+```go 
+    var mongoUri = 
+      "<Your Atlas Connection String>";
+```
+
+### 4. Run the Project
+
+If you are running from the shell, you can run the application from the
+directory that contains it with the following command:
+
+```shell
+go run main.go
+```
+
+If you are running it from the IDE, use the appropriate command to run the
+contents of the `main.go` file.
+
+1. Click the Run icon, or from the Run menu, choose **Start Debugging**.
```

Comment:
The reason I asked about copypasta is that these instructions refer to some specific IDE such as Visual Studio or VSCode. This specific menu may not be available depending on what they're using (even if you assumed VSCode, there are multiple golang plugins)

Reply:
What would you recommend updating this text with? I'm using the official go plugin by google for vscode and can run it using the run button on the left hand side of the editor